src/Uno.UI/UI/Xaml/Markup/Reader/XamlObjectBuilder.cs

```diff
@@ -725,24 +727,37 @@ private void ProcessStaticResourceMarkupNode(object instance, XamlMemberDefiniti
 			{
 				var keyName = resourceNode.Members.FirstOrDefault()?.Value?.ToString();
 				var dependencyProperty = TypeResolver.FindDependencyProperty(member);
-
 				if (keyName != null
 					&& dependencyProperty != null
 					&& instance is DependencyObject dependencyObject)
 				{
-					ResourceResolver.ApplyResource(
-						dependencyObject,
-						dependencyProperty,
-						keyName,
-						isThemeResourceExtension: IsThemeResourceMarkupNode(member),
-						isHotReloadSupported: true);
-
-					if (instance is FrameworkElement fe)
+					if (IsCustomResourceMarkupNode(member))
+					{
+						string objectType = dependencyObject.GetType().Name;
+						string propertyName = dependencyProperty.Name;
+						string propertyType = dependencyProperty.Type.Name;
```

Comment:
```suggestion
						string objectType = dependencyObject.GetType().FullName;
						string propertyName = dependencyProperty.Name;
						string propertyType = dependencyProperty.Type.FullName;
```

Based on UWP's behavior:

![image](https://user-images.githubusercontent.com/5839577/187913963-db67fa8c-6621-43b7-bb4f-e27112a72cf4.png)